dapr/clients/http/dapr_actor_http_client.py

```diff
@@ -47,7 +49,14 @@ async def invoke_method(
         """
         url = f'{self._get_base_url(actor_type, actor_id)}/method/{method}'
 
-        body, r = await self._client.send_bytes(method='POST', url=url, data=data)
+        # import to avoid circular dependency
+        from dapr.actor.runtime.reentrancy_context import reentrancy_ctx
+        reentrancy_id = reentrancy_ctx.get()
+        headers: Dict[str, Union[bytes, str]] = (
+            {DAPR_REENTRANCY_ID_HEADER: reentrancy_id} if reentrancy_id else {})
```

Comment:
Can this erase other headers that were already set/needed? Or is this the only header? I'm thinking of things like content-type or length.

Reply:
Previously no custom headers were sent at all. That hasn't been supported with Actors in Python. The `invoke_method` only supports a single payload that is byte-encoded. This also only returns bytes.

The behavior prior to this PR is as follows:
Invoke_method always sends an empty dictionary for the headers via `send_bytes()`. Then that function checks whether the Content Type and DAPR API Token headers are presented (which they aren't) and if not uses defaults. See the `send_bytes` method definition and the test modification I made.

This introduces no breaking change.